### ISSUE TITLE
gh2md --multiple-files should produce canonical file paths

### ISSUE THREAD

**Issue by milahu** [mattduck/gh2md#45]
actual

```
2020-11-04.3.issue.closed.md
2022-07-09.24.issue.open.md
```

problem: `closed` and `open` are not "canonical" but variable
this means issue urls are not permalinks, and git history is broken at file renames

expected

```
2020-11-04.3.issue.md
2022-07-09.24.issue.md
```

or

```
issues/3.md
issues/24.md
```

**Comment by mattduck** [comment]
Yes agree -- have no idea why I did this this way. Feel free to make a PR, I think for backwards compatibility initially it'd make sense to have this be optional behaviour controlled by a flag.
